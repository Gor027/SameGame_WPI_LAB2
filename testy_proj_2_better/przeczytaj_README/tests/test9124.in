XXU XU   X    UXU
X X X  U U UU UX   X XUU UUXXUXUXU XX    UUUX UX XUU
 XXXXXX U  U  UXXUX   XUX UXX U U    XU  XUX  UXXUX X UXUUXUU X X UX X U
X  XXXXUXX  X U X X X   U  XUXX  X  X UU   X UX 
XU XX UXU  XXUXXX UXXUXXUX  XU X  UXU UXUXUUUXU    XX X     UUXXUX
X   UXX UXXUX XXX X  XU U U X X UX X  UU  UX  X  U XUU XUUXUXUX XX
X X U XX  U UUUUXXXXXUXUUX XU     U  XX XUUUXXU  
XXUX  UU  UUUXXUXX XU UXXU  UU U  XU UXUXUU U XX XU
X XX  XXUUUUXU  X XXX U U XU XU        XXXUU UU XU UXUUUUX XUX XX 
 UUXU  XUXXXUUXXUUUX U 
 XU X  X
UXUX UUX  UXUUUX U UXU X  X UX   UX XU UXXU UX U U     UU UXXXXX UUU XX U  XXU 
UXXU X XX UXXU U UXU UXUU UU XXUXU  UU UXUUX U U X 
UXX X XXUXU UXU UUUUUX UUXXXX XUXXU  X XU  X XXXUX XXU X UU UXXXXU 
