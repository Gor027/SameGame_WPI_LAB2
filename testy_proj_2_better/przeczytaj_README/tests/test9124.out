  U    U    U U
  X X  U    X UXU
 XX X XX   UX UXX XX    X   X             UU  U
XXXXX XXU XUXUUXX XX XUUU   X          U UUX  U  X
XUUXX UUXXUXUUXXXUUXXUXUU UXUUX        XUXUX UX  U
XXXXXUXXUUUUUXXXXXXXXXXXU UXXXU  U   XUUXXUU XUX U          U
XUUXUXXXUXUUXUUUXXXXXUUUUUUXXUXUXXU XXUUXUUXXXXX X UX  X    UUX X
XXUXUXUXUXXUUUXUXUXXUUXXXXXXUUXXUXUXXUUXXUUUUUUX U XX  XUUXUXUXXUX
XXUXUXXXUUUXUUXXXUUUXUUXUXXUUXUUUUXXUUUUXXUUXXXUXUUXUUUUUUXUUXXXXX
XXXXXUUXXXUXXUUXUUXUXUUUUUUUUXXUXUXXXUXUXXUXUUUUXXXUXUXUUXUXXXXXXUUXX
XXXUXXXXUXUXUXUUUUUUUXXUUXXXXXXUXXUUUXXXUUUXUXXXUXUXXUUXUUUXUXXXXUUUXXUUXXU
